src/Abuse/Adapters/ReCaptcha.php

```diff
@@ -79,4 +79,28 @@ public function check()
 
         return $result['success'];
     }
+
+    /**
+     * Delete logs older than $seconds seconds
+     * 
+     * @param int $seconds 
+     * 
+     * @return bool   
+     */
+    public function deleteLogsOlderThan(int $seconds):bool
```

Comment:
Throw an exception saying this methods are not supported for the Recaptch adapter.

Reply:
Done